export-script/generate_assertion_details.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# This script generates assertion details from the PostgreSQL database and saves them to JSON files.
+# The script is designed to run on a local machine and requires the following:
+# - psql (PostgreSQL client) installed on the local machine
+# - Access to the PostgreSQL database (host, name, user, and password)
+
+# Prompt the user for the PostgreSQL password
+read -s -p "Enter the PostgreSQL password: " PG_PASSWORD
+echo
+
+DB_HOST="corpus-dev-dump-2024-05-06.cpcwgoa3uzw1.eu-west-1.rds.amazonaws.com"
```

Comment:
We can also make this an input we get from user since this will change very often